frontend/src/components/modals/modalparts/TaskMapInfoModalColumns.tsx

```diff
@@ -10,6 +10,7 @@ import ZoomInIcon from '@mui/icons-material/ZoomIn';
 export const overviewColumns: {
   Title: FC;
   columns: { Subtitle: FC; Description: FC; Icon?: FC }[];
+  key: string;
 }[] = [
   {
     Title: () => <Trans i18nKey="Task relations" />,
```

Comment:
The another option for the key is to use the title. Sorry I did not explain it coherently previously.
```suggestion
    title: "Task relations",
```
Then title should be used as following `<Trans i18nKey={title} />` instead of `<Title />`
For actionIcons, the Action could be used similarly